examples/classification/plot_digits_classification.py

```diff
@@ -102,3 +102,25 @@
 print(f"Confusion matrix:\n{disp.confusion_matrix}")
 
 plt.show()
+
+###############################################################################
+# We can also build a :func:`~sklearn.metrics.classification_report`
+# through the :ref:`confusion matrix <confusion_matrix>`.
```

Comment:
```suggestion
# If the results from evaluating a classifier are stored in the form of a
# :ref:`confusion matrix <confusion_matrix>` and not in terms of `y_true` and
# `y_pred`, one can still build a :func:`~sklearn.metrics.classification_report`
# as follows:
```

Reply:
I really like this one, thanks for the suggestion!